bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -216,6 +221,29 @@
 		</event>
 	</channel-type>
 
+	<channel-type id="protectionPlusEvent">
+		<kind>trigger</kind>
+		<label>Protection Plus Event</label>
+		<description>Triggered when a Protection Plus capable device detects a break-in attempt.</description>
+		<event>
+			<options>
+				<option value="ALARM">alarm</option>
```

Comment:
A detzil but you could put the first letter of your label in uppercase like for the other triggers.